From ec2605216c5d218220be24cbbcf02f1873957de1 Mon Sep 17 00:00:00 2001
From: Firoz Khan <firoz.khan@linaro.org>
Date: Sun, 29 Jul 2018 11:52:54 +0530
Subject: [PATCH v4 1/5] m68k: rename system call table file name

The system call table entries are present in syscalltable.S.
One of the patch in this patch series will auto-generate the
system call table entries using the system call table gener-
ation script and which will replace the existig table file.

In order to come up with a common naming convention across
all architecture, we need to change the file name to syscall-
_table.S. This change will unify the implementation across
all architecture.

Signed-off-by: Firoz Khan <firoz.khan@linaro.org>
---
 arch/m68k/kernel/Makefile                            | 2 +-
 arch/m68k/kernel/{syscalltable.S => syscall_table.S} | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename arch/m68k/kernel/{syscalltable.S => syscall_table.S} (99%)

diff --git a/arch/m68k/kernel/Makefile b/arch/m68k/kernel/Makefile
index dbac7f8..5f40ce6 100644
--- a/arch/m68k/kernel/Makefile
+++ b/arch/m68k/kernel/Makefile
@@ -15,7 +15,7 @@ extra-$(CONFIG_SUN3)	:= sun3-head.o
 extra-y			+= vmlinux.lds
 
 obj-y	:= entry.o irq.o module.o process.o ptrace.o
-obj-y	+= setup.o signal.o sys_m68k.o syscalltable.o time.o traps.o
+obj-y	+= setup.o signal.o sys_m68k.o syscall_table.o time.o traps.o
 
 obj-$(CONFIG_MMU_MOTOROLA) += ints.o vectors.o
 obj-$(CONFIG_MMU_SUN3) += ints.o vectors.o
diff --git a/arch/m68k/kernel/syscalltable.S b/arch/m68k/kernel/syscall_table.S
similarity index 99%
rename from arch/m68k/kernel/syscalltable.S
rename to arch/m68k/kernel/syscall_table.S
index 2c8402e..6145bc0 100644
--- a/arch/m68k/kernel/syscalltable.S
+++ b/arch/m68k/kernel/syscall_table.S
@@ -6,7 +6,7 @@
  *
  *  Copyright (C) 1998  D. Jeff Dionne <jeff@lineo.ca>,
  *                      Kenneth Albanowski <kjahds@kjahds.com>,
- *  Copyright (C) 2000  Lineo Inc. (www.lineo.com) 
+ *  Copyright (C) 2000  Lineo Inc. (www.lineo.com)
  *  Copyright (C) 1991, 1992  Linus Torvalds
  *
  *  Linux/m68k support by Hamish Macdonald
-- 
1.9.1

